### DISCUSSION TITLE
What does meow-reverse do?

### DISCUSSION THREAD

**Discussion by Esnos33** [meow-edit/meow#566] [Q&A] [answered]
Hi, I done tutorial yesterday, and I still can't understand what does `meow-reverse` (`;` on qwerty layout) does. I checked explanation with `C-h k` and in [COMMAND.ORG](https://github.com/meow-edit/meow/blob/master/COMMANDS.org#meow-reverse) and searched discussions for `meow-reverse` but nothing helped me.

**Comment by Haxxflaxx** [comment, marked as answer]
Some commands in meow have a inherent direction such as `meow-next-word` etc... Other commands can run both forward or backward, such as `meow-search` or `meow-line`. These commands can be changed direction of by calling `meow-reverse`.

**Reply by Haxxflaxx** [reply]
As an example you can try to press the keys "xx;xx;xx" while in the middle of a list to see what it does.

**Reply by Esnos33** [reply]
I understand now, thanks for answer. I somehow missed this behavior in `meow-tutor` example.
